stories/components/form.stories.tsx

```diff
@@ -28,16 +20,21 @@ export function Control() {
   const [radio, setRadio] = useState<RadioOption | undefined>(undefined);
   return (
     <ExampleGroup style={{ display: 'inline-block' }}>
-      <Field label="Name" name="name" required>
-        <Input type="text" maxLength={10} />
-      </Field>
-      <Field label="Email" name="email">
-        <Input type="email" />
-      </Field>
-      <Field label="date of birth" name="bd">
-        <Input type="date" />
-      </Field>
-      <Field label="Status" name="status">
+      <FormGroup
+        label="Name"
+        labelFor="name"
+        labelInfo={<span style={{ color: 'red' }}>*</span>}
+        intent="danger"
+      >
+        <InputGroup type="text" maxLength={10} />
+      </FormGroup>
```

Comment:
Forms inputs have missing `id` attributes to link with `labelFor` and enable focus of input when clicking the label.

Please check all stories for this.